tests/baselines/reference/jsContainerMergeTsDeclaration2.symbols

```diff
@@ -1,18 +1,18 @@
 === tests/cases/conformance/salsa/a.d.ts ===
 declare namespace C {
->C : Symbol(C, Decl(a.d.ts, 0, 0), Decl(b.js, 0, 0))
+>C : Symbol(C, Decl(a.d.ts, 0, 0), Decl(b.js, 0, 0), Decl(b.js, 0, 17))

```

Comment:
so, before only `declare namespace C { }` and `C.prototype = {}` functioned as declarations. Now `C.bar = 2` does too. Why didn't this before?

Reply:
`C.bar` saw `C` was already bound and declined to add `C` to `C`'s declarations (which simultaneously prevented `C` in `C.bar` from having a symbol!). Now we always add the new declaration.